ibis/backends/mysql/registry.py

```diff
@@ -242,6 +255,8 @@ def compiles_mysql_trim(element, compiler, **kw):
         ops.Strip: unary(lambda arg: _mysql_trim(arg, "both")),
         ops.LStrip: unary(lambda arg: _mysql_trim(arg, "leading")),
         ops.RStrip: unary(lambda arg: _mysql_trim(arg, "trailing")),
+        ops.TimeDelta: _temporal_delta,
+        ops.DateDelta: _temporal_delta,
```

Comment:
MySQL has a `timestampdiff` function, but it truncates after compute the difference and there's no `date_trunc` function to manually truncate prior to diffing.

This could be done with date formatting, but that can be done in a follow up.